src/Streamly/Core.hs

```diff
@@ -150,10 +150,10 @@ data SVar m a =
 --
 -- data Stream m a = Stop | Yield a (Maybe (Stream m a))
 --
--- Our goal is to be able to represent finite as well infinite streams and
+-- Our goal is to be able to represent finite as well as infinite streams and
 -- being able to compose a large number of small streams efficiently. In
 -- addition we want to compose streams in parallel, to facilitate that we
--- maintain a local state in an SVar that is shared across and is used for
+-- maintain a local state in a SVar that is shared across and is used for
```

Comment:
I pronounce this as "ess-var", so based on the sound "an" sounds appropriate :-)

Reply:
I just saw that in many place, was going to cancel that change :P

Reply:
BTW, this documentation is not user visible. Though it should be correct nevertheless, but we do not have to as stringent as we have to be for user visible docs.

Reply:
Contributors are users :P, we learn your architecture by reading the comments aswell ^^